src/ccid.h

```diff
@@ -194,6 +194,37 @@ typedef struct
 #define PROTOCOL_ICCD_A	1	/* ICCD Version A */
 #define PROTOCOL_ICCD_B	2	/* ICCD Version B */
 
+/* Known CCID bMessageType values. */
```

Comment:
Do not use `enum` if it makes no sens.
I prefer `#define` in this case since all the values are hardcoded.